tests/testAutopas/tests/tuning/utils/SmoothingTest.cpp

```diff
@@ -21,7 +22,7 @@ TEST(SmoothingTest, lowessLastPoint) {
 
   // YS values with F = .25, NSTEPS = 0, DELTA = 0.0
   {
-    auto out = autopas::smoothing::smoothLastPoint(obs, (.25 * obs.size()));
-    EXPECT_EQ(out, 6);
+    auto out = autopas::smoothing::smoothLastPoint(obs, (size_t)(.25 * (double)obs.size()));
```

Comment:
Is there are reason you're using the C-style cast instead of C++'s `static_cast`?

Reply:
Nope, I just forgot to use it here. Updated in 7b40e0d3